__tests__/app.test.js

```diff
@@ -73,26 +74,56 @@ describe("GET articles", () => {
       });
   });
 
+  test("400: Responds with 'Bad Request' for invalid article id", () => {
+    return request(app)
+        .get("/api/articles/not-a-valid-id")
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request");
+        });
+});
+
+test("404: Responds with 'Not Found' for non-existent article id", () => {
+    return request(app)
+        .get("/api/articles/999999")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Not Found");
+        });
+});
+});
 
-describe("POST endpoints", () => {
-  test("201: Responds with new data", () => {
-    const newDescription = {
-      description: "serves an object of specific article by id",
-      queries: ["article_id"]
-    };
 
-  return request(app)
-    .post("/api")
-    .send(newDescription)
-    .set('Content-Type', 'application/json')
-    .expect(201)
-    .then(({body}) => {
-      expect(body).toMatchObject({
-        description: expect.any(String),
-        queries: expect.any(Array),
-        exampleResponse: expect.any(Object),
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body;
+        expect(articles).toBeSortedBy("created_at", {descending: true});
+        expect(articles).toHaveLength(13); 
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number)
+          });
+        });
       });
-    });
   });
+
+  test("400: Responds with 'Bad Request' for invalid endpoind", () => {
+    return request(app)
+        .get("/api/article")
+        .expect(404)
+        .then(({body}) => {
+          expect(body.msg).toBe("Route Not Found")
+      })
```

Comment:
a fine test, but not necessarily tied to api/articles SAD path. It's more a general test.  Perhaps put it under its own describe block.